### ISSUE TITLE
Will the code for Mobilenet V3 be released?

### ISSUE THREAD

**Issue by wi11iamzhao** [tensorflow/tpu#522]
I want to try to use [Mobilenet V3](https://arxiv.org/abs/1905.02244) on some tasks, but there seems to be no official implementation.I know that the architecture of Mobilenet V3 is very similar to MNASNet. But when I try to implement it, I find that the current version of Tensorflow (r1.15rc/r2.0.0rc2) does not seem to provide a hard swish activation function.The operator HardSwish can be found in the source code of TF Lite, but the python interface is not found and no documentation is found.
I know that [MixNet](https://arxiv.org/abs/1907.09595) seems to achieve higher ILSVRC accuracy with less computational effort.However, the latency data is not available in MixNet's paper, and MobileNet V3 offers a smaller model selection (Mobilenet V3 Small).
So, I would like to ask if you will release the Mobilenet V3 code and pre-train checkpoints.Or can you provide a way to call the hard swish activation function?

**Comment by saberkun** [comment]
For TF Lite questions, please raise an issue in tensorflow repo.
We will consider MobileNetV3 in TF2.

**Comment by wi11iamzhao** [comment]
Thank you for your reply, I will close this issue.
